crates/red_knot_python_semantic/resources/mdtest/type_properties/is_subtype_of.md

```diff
@@ -0,0 +1,232 @@
+# Subtype relation
+
+## Basic types
+
+```py
+from typing import Any
+from typing_extensions import Literal
+from knot_extensions import Unknown, is_subtype_of, static_assert
+
+static_assert(is_subtype_of(str, object))
+static_assert(is_subtype_of(int, object))
+static_assert(is_subtype_of(bool, object))
+
+static_assert(is_subtype_of(bool, int))
+static_assert(is_subtype_of(TypeError, Exception))
+static_assert(is_subtype_of(FloatingPointError, Exception))
+
+static_assert(not is_subtype_of(object, int))
+static_assert(not is_subtype_of(int, str))
+static_assert(not is_subtype_of(int, Literal[1]))
+
+static_assert(not is_subtype_of(Unknown, Unknown))
+static_assert(not is_subtype_of(Unknown, Literal[1]))
+static_assert(not is_subtype_of(Literal[1], Unknown))
+
+static_assert(not is_subtype_of(Any, Any))
+static_assert(not is_subtype_of(Any, Literal[1]))
+static_assert(not is_subtype_of(Literal[1], Any))
```

Comment:
Maybe we can move those tests involving `Any` or `Unknown` into a new subsection titled "Non fully-static types do not participate in subtyping"?